client/js/models/archive.js

```diff
@@ -205,7 +209,7 @@ module.exports = {
       }
       if (!archive) {
         archive = drive.createArchive(key)
-        sw = swarm(archive)
+        sw = swarm(archive, {signalhub: state.signalhubs})
```

Comment:
does this still work with the dat.land signalhub added here? I had some trouble locally trying this yesterday, but unclear if it was due to the extra signalhub. Is it connecting reliably for you?